en/components/calendar.md

```diff
@@ -4,40 +4,190 @@ _description: Create an intuitive calendar for applications to display date info
-<div class="sample-container loading" style="height:1728px">
-    <iframe id="calendar-sample-iframe" src='{environment:demosBaseUrl}/calendar' width="100%" height="100%" seamless frameBorder="0" onload="onSampleIframeContentLoaded(this);"></iframe>
+<div class="sample-container loading" style="height: 500px">
+    <iframe id="calendar-sample-5-iframe" src='{environment:demosBaseUrl}/calendar-sample-5' width="100%" height="100%" seamless frameBorder="0" onload="onSampleIframeContentLoaded(this);"></iframe>
 </div>
 <div>
-    <button data-localize="stackblitz" class="stackblitz-btn" data-iframe-id="calendar-sample-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
+    <button data-localize="stackblitz" class="stackblitz-btn" data-iframe-id="calendar-sample-5-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
 </div>
-<div class="divider--half"></div>
+
+> [!WARNING]
+> Ignite UI for Angular has a [peer dependency](https://nodejs.org/en/blog/npm/peer-dependencies/) on [**HammerJS**](https://hammerjs.github.io/).
+> Make sure you add it to your Angular project.
+> <br/> <br/>`$ npm install hammerjs @types/hammerjs`
 
 ### Usage
+To get started with the **Ignite UI for Angular Calendar**, let's first import the **IgxCalendarModule** in the application's AppModule, typically this is the **app.module.ts** file. Note that the **IgxCalendar** is also dependent on the **BrowserAnimationsModule**, so it needs to be added to the AppModule as well:
+
 ```typescript
-import { IgxCalendarComponent } from "igniteui-angular";
+// app.module.ts
+...
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { IgxCalendarModule } from 'igniteui-angular/main';
+@NgModule({
+    ...
+    imports: [..., BrowserAnimationsModule, IgxCalendarModule],
+    ...
+})
+export class AppModule {}
+```
+You will usually also import the **IgxCalendarComponent** in the AppComponent file (or your editor will auto-import them for you) when declaring types that are part of the calendar API:
+.
+
+```typescript
+import { IgxCalendarComponent } from 'igniteui-angular/main';
+...
+
+@ViewChild('calendar', { read: IgxCalendarComponent }) public calendar: IgxCalendarComponent;
 ```
 
-Basic initialization
+> [!WARNING]
+> Note that the **igxCaledarComponent** uses the [Intl](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/DateTimeFormat) WebAPI for localization and formatting of dates. 
+> Consider using the [appropriate polyfills](https://github.com/andyearnshaw/Intl.js/) if your target platform does not support them.
+
+### Selection
+Instantiating the **IgxCalendarComponent** is as easy as placing its selector element in the template. This will leave the calendar display the current month by default and run in single selection mode. We switch to any of the other selection modes - **multi** and **range**, by setting value to the **selection** property:
```

Comment:
This will display the current month in the calendar and use single selection mode.
by setting the **selection** property